README.md

```diff
@@ -81,3 +82,28 @@ Make sure the service account has the correct permissions to deploy the desired
 
 * roles/cloudfunctions.admin
 * roles/iam.serviceAccountUser
+
+
+## Requirements file
```

Comment:
requirementx.txt is not required if poetry is being used. we need one or the other in order to install the dependencies for deployment